packages/cache/src/internal/tar.ts

```diff
@@ -61,14 +61,21 @@ export async function extractTar(
   // Create directory to extract tar into
   const workingDirectory = getWorkingDirectory()
   await io.mkdirP(workingDirectory)
-  // --d: Decompress.
+  // -d: Decompress.
+  // unzstd is equivalent to 'zstd -d'
   // --long=#: Enables long distance matching with # bits. Maximum is 30 (1GB) on 32-bit OS and 31 (2GB) on 64-bit.
   // Using 30 here because we also support 32-bit self-hosted runners.
   function getCompressionProgram(): string[] {
     switch (compressionMethod) {
       case CompressionMethod.Zstd:
+        if (process.platform === 'win32') {
```

Comment:
May want to use a const to keep this simpler to read
`const IS_WINDOWS = process.platform === 'win32'`